package.json

```diff
@@ -287,7 +287,7 @@
         },
         {
           "command": "java.view.package.exportJar",
-          "when": "view == javaProjectExplorer && viewItem =~ /java:workspace.*?\\+uri/ && java:serverMode!= LightWeight",
+          "when": "view == javaProjectExplorer && workspaceFolderCount != 0 && viewItem =~ /java:workspace.*?\\+uri/ && java:serverMode!= LightWeight",
```

Comment:
Seems no need to have `workspaceFolderCount != 0`. Because if there is no workspace, this node won't appear in explorer